database_scripts/PR1347/apply.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE questionnaire_state
+  ADD COLUMN created INT;
+
+ALTER TABLE questionnaire_state
+  ADD COLUMN last_updated INT;
+
+ALTER TABLE questionnaire_state
+  ADD COLUMN version INT;
```

Comment:
- Directory names should be hyphen separated 
- I think we can drop a directory level, so we'd have `database-scripts/pr-1347-apply.sql`